pimd/pim_cmd.c

```diff
@@ -1333,7 +1333,7 @@ static void igmp_show_statistics(struct pim_instance *pim, struct vty *vty,
 		json_object_int_add(json_row, "queryV1", rx_stats.query_v1);
 		json_object_int_add(json_row, "queryV2", rx_stats.query_v2);
 		json_object_int_add(json_row, "queryV3", rx_stats.query_v3);
-		json_object_int_add(json_row, "leaveV3", rx_stats.leave_v2);
+		json_object_int_add(json_row, "leaveV2", rx_stats.leave_v2);
```

Comment:
The change is right but won't it be causing script failures in customer env if they are using this one.

Reply:
IMO, if it causes failures, then the operator already failed by looking for V2 leave messages :smile:

Reply:
I'm with @ton31337 , this is flat out wrong. this is different from cases were we had typo fixes in json where we don't want to break existing code because actual returned data is wrong. If someone is using it as-is it might be better to break it so that they can fix it.

Reply:
@mobash-rasool Is this resolved?

Reply:
Yeah approved.